doc/jetty-config.md

```diff
@@ -39,6 +39,16 @@ that exceeds this value, Jetty will return an HTTP 413 Error response. This
 defaults to 8192 bytes, and only needs to be configured if an exceedingly large
 header is being sent in an HTTP Request.
 
+### `so-linger-seconds`
+
+This sets the TCP SO_LINGER time, in seconds, that the webserver uses for
+underlying socket connections.  Values less than 0 result in SO_LINGER
+being disabled.  Defaults to -1, i.e., "disabled".  For a more detailed
+description of what it means to have SO_LINGER disabled vs. enabled for some
+number of seconds, see http://man7.org/linux/man-pages/man7/socket.7.html.  Note
+that the effect of setting this option may vary depending upon the operating
+system's underlying implementation.
+
```

Comment:
This may only be used in rare circumstances.  I hesitate to say anything more sweeping like "if you have a load balancer routing traffic to your web server and your webserver is leaking file descriptors, this could clean them up".  What if I added something like this?

*You may consider enabling SO_LINGER if non-well behaving clients prevent the webserver from gracefully closing out server sockets properly.  This should be considered more of a "last resort", however.  In such scenarios, it would be better to find a way for clients to terminate socket connections more gracefully.*

@cprice404 - what do you think?